config/jobs/kubernetes/sig-arch/kubernetes-depstat.yaml

```diff
@@ -0,0 +1,24 @@
+presubmits:
+  kubernetes/kubernetes:
+  - name: kubernetes-depstat
+    decorate: true
+    always_run: true
+    skip_report: true
+    spec:
+      containers:
+      - image: golang
+        command: ["/bin/bash"]
+        args: ["-c", "set -euo; 
```

Comment:
I think one of the [tests is failing](https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/test-infra/22137/pull-test-infra-bazel/1392514890026979328#1:build-log.txt%3A6289) because it expects a `]` at the end of this line. So should I have the entire thing in one line to fix this? '-'

@dims @spiffxp

Reply:
See the other pattern in general use : https://cs.k8s.io/?q=-%20%5C%7C&i=nope&files=&excludeFiles=&repos=kubernetes/test-infra